$OpenBSD: patch-src_shares_shares-tool_c,v 1.2 2011/01/17 17:51:25 ajacoutot Exp $

XXX We never display NFS even if it's part of OpenBSD base system
because the system-tools-backends use the Linux|SunOS exports format.

--- src/shares/shares-tool.c.orig	Mon Jan 17 17:30:37 2011
+++ src/shares/shares-tool.c	Mon Jan 17 17:32:58 2011
@@ -151,7 +151,11 @@ check_servers (GstSharesTool *tool)
 					 GTK_BUTTONS_CLOSE,
 					 _("Sharing services are not installed"));
 	gtk_message_dialog_format_secondary_markup (GTK_MESSAGE_DIALOG (dialog),
+#ifndef __OpenBSD__
 						    _("You need to install at least either Samba or NFS "
+#else
+						    _("You need to install Samba "
+#endif
 						      "in order to share your folders."));
 	gtk_dialog_run (GTK_DIALOG (dialog));
 	gtk_widget_destroy (dialog);
@@ -221,8 +225,10 @@ gst_shares_tool_update_services_availability (GstShare
 
 		if (role == GST_ROLE_FILE_SERVER_SMB)
 			tool->smb_available = TRUE;
+#ifndef __OpenBSD__
 		else if (role == GST_ROLE_FILE_SERVER_NFS)
 			tool->nfs_available = TRUE;
+#endif
 
 		g_object_unref (service);
 		valid = oobs_list_iter_next (services, &iter);
